File: 1165.py
n = input()
for i in range(0, n, 1):
	primo = True
	teste = input()
	for x in range(2, teste, 1):
		if (teste % x == 0):
			primo = False
	if (primo == True):
		print "%d eh primo"%(teste)
	else:
		print "%d nao eh primo"%(teste)

File: 1165.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n, x, cont = 0;
		n = sc.nextInt();
		boolean[] primo = new boolean[n];
		int[] teste = new int[n];
		for (x = 0; x < n; x++) {
			primo[x] = true;
			teste[x] = sc.nextInt();
			for (int i = 2; i <= teste[x]; i++) {
				if ( ( (teste[x] % i) == 0) && (i != teste[x]) ) {
					primo[x] = false;
				}
			}
		}
		for (x = 0; x < n; x++) {
			if (primo[x]) {
				System.out.printf("%d eh primo%n", teste[x]);
			} else {
				System.out.printf("%d nao eh primo%n", teste[x]);
			}
		}
	}
}
